Users/Docker_Image.rst

```diff
@@ -9,7 +9,7 @@ on your code in the working directory, like this:
 
 ::
 
-    docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
+  docker run --volume=$(pwd):/app --workdir=/app coala/base coala --ci
```

Comment:
`Users/Docker_Image.rst
|  66| ••••script:•docker•run•--volume=$(pwd):/app•--workdir=/app•coala/base•coala•--ci
|    | [NORMAL] LineLengthBear:
|    | Line is longer than allowed. (80 > 79)
`
As on adding coala --ci in place of coala-ci , I got this, that'why I have removed spaces.
Yes, I can do  another thing also by continuing in next line like
`docker run --volume=$(pwd):/app --workdir=/app coala/base \
    coala --ci`